src/backend/distributed/shared_library_init.c

```diff
@@ -327,6 +361,18 @@ RegisterCitusConfigVariables(void)
 		NULL, NULL, NULL);
 
 	DefineCustomBoolVariable(
+		"citus.log_distributed_deadlock_detection",
+		gettext_noop("Log distributed deadlock detection related processing in "
+					 "the server log"),
```

Comment:
As I wrote somewhere else, I'm less concerned about the guc, then about rotting code.